README.md

```diff
@@ -1,2 +1,41 @@
 # Hello_Microverse_Alex
-Test Microverse Repo implementing awesome features! Writen by Alexander Saavedra
+Test Microverse repository is implementing awesome features! Writen by Alexander Saavedra
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Test_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- [ ] You did great using the provided Microverse readme template. :+1: :+1: Looking at the readme file, it seems you have removed most of the important sections and also not customized it to your project. As the previous reviewer suggested and also According to Microverse the only optional section is the FAQ, you are meant to customize every other section in your readme file. :smile: :wave: I'll suggest you get the [template once more,](https://github.com/microverseinc/readme-template) make sure you do not remove any section. Then go through your readme and remove the placeholder logo and texts, and then fill the content of every section in the readme file :+1: :100: 

**THIS IS AN EXAMPLE OF HOW THE REAMDE SHOULD LOOK LIKE FOR THIS PROJECT. [LINK](https://github.com/enocol/HelloMicroverse/blob/newbranch/README.md)**